app/pandas.ipynb

```diff
@@ -51,24 +52,90 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "data = {\n",
     "    \"Name\": [\"John\", \"Emily\", \"Ryan\"],\n",
     "    \"Age\": [16, 28, 22],\n",
     "    \"City\": [\"New York\", \"Los Angeles\", \"Chicago\"],\n",
     "}\n",
     "\n",
-    "# write your code here"
+    "df = pd.DataFrame({\n",
+    "    \"Name\": [\"John\", \"Emily\", \"Ryan\"],\n",
+    "    \"Age\": [16, 28, 22],\n",
+    "    \"City\": [\"New York\", \"Los Angeles\", \"Chicago\"],\n",
+    "})\n",
```

Comment:
The DataFrame is created directly from the dictionary values instead of using the given 'data' dictionary. Consider using 'df = pd.DataFrame(data)' to utilize the provided dictionary.